.github/workflows/test-release.yml

```diff
@@ -69,9 +69,60 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.7, 3.8, 3.9]
-        os: [ubuntu-latest, macos-latest, windows-latest]
-        dist: [wheel, sdist]
```

Comment:
'dist' might still be needed

Reply:
It's in the previous matrix that I'm putting back, and it's used by the `- name: Install wheel` and `- name: Install sdist` steps